packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -14,6 +16,23 @@ const NominationBanner = ({ nomination }) => {
   const time = newDate.toLocaleDateString();
   const minutes = newDate.toLocaleTimeString([], { hour: '2-digit', minute: '2-digit' });
   const finalDate = `${time} – ${minutes}`;
+  const [activeNomination, setActiveNomination] = useContext(ActiveNominationContext);
+
+  function declineApplication() {
+    const declineStatus = 'Ready for Board Review'
+    activeNomination.status = declineStatus
+    setActiveNomination({ ...activeNomination })
```

Comment:
Minor, but these two lines can be combined to `setActiveNomination({...activeNomination, status: declineStatus})`

Reply:
I just tested this in browser, it looks like the statuses element on the page does not get updated on click when it is written as one line, if it is written like that I have to refresh the page to see the statues element react to the on click.